rosdep/base.yaml

```diff
@@ -2037,9 +2037,11 @@ libnl-dev:
 libnlopt-dev:
   fedora: [NLopt-devel]
   ubuntu: [libnlopt-dev]
+  debian: [libnlopt-dev]
```

Comment:
The `debian` key needs to be inserted in alphabetical order (hence above `fedora`).

Same below.